<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dynamic</class>
 <widget class="QWidget" name="dynamic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>428</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>По</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>С</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Группы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Дисциплины</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
     background-color: pink;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: black;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
 }
 QPushButton:pressed {
     background-color: rgb(0, 255, 238);
     border-style: inset;
 }</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
